rskj-core/src/main/java/co/rsk/peg/utils/BtcTransactionFormatUtils.java

```diff
@@ -57,4 +68,38 @@ public static boolean isBlockHeaderSize(int size, ActivationConfig.ForBlock acti
         return (activations.isActive(ConsensusRule.RSKIP124) && size == MIN_BLOCK_HEADER_SIZE) ||
                 (!activations.isActive(ConsensusRule.RSKIP124) && size >= MIN_BLOCK_HEADER_SIZE && size <= MAX_BLOCK_HEADER_SIZE);
     }
+
+    public static byte[] extractOpReturnData(BtcTransaction btcTx) throws NoOpReturnException, InvalidOpReturnOutputException {
+        byte[] data = new byte[]{};
+        int opReturnOccurrences = 0;
+
+        logger.info("[getOpReturnOutput] Getting OP_RETURN data for btc tx: {}", btcTx.getHash());
+
+        for (int i=0; i<btcTx.getOutputs().size(); i++) {
+            List<ScriptChunk> chunksByOutput = btcTx.getOutput(i).getScriptPubKey().getChunks();
+            if (chunksByOutput.get(0).opcode == ScriptOpCodes.OP_RETURN) {
+                if (chunksByOutput.size() > 1) {
+                    data = btcTx.getOutput(i).getScriptPubKey().getChunks().get(1).data;
+                    opReturnOccurrences++;
+                } else {
+                    opReturnOccurrences++;
+                    data = null;
+                }
+            }
+        }
+
+        if (opReturnOccurrences == 0) {
+            String message = "No OP_RETURN output found for tx";
+            logger.debug("[getOpReturnOutput] {}", message);
```

Comment:
```suggestion
            logger.debug("[extractOpReturnData] {}", message);
```
And there is another logger with the same typo